index.html

```diff
@@ -930,6 +942,61 @@ <h2>
         evolving privacy landscape and participate in the corresponding
         evolution of the API.
       </p>
+      <section>
+        <h3>
+          Design Considerations and Alternatives
+        </h3>
+        <p>
+          The Digital Credentials API is designed to mediate digital credential
+          requests from websites, being agnostic to the credential format and
+          the information contained in it, as well as the protocol used to
+          exchange it (within the bounds on the protocol registry inclusion
+          criteria). This and other key design choices are derived from the
+          goal of providing a more secure and private credential exchange
+          experience for users than the existing technological alternatives,
+          that is still compatible with the underlying protocols for ease of
+          adoption.
+        </p>
+        <p>
+          The API provides the connection interface between [=verifiers=] and
+          [=holders=], i.e. the means by which a [=digital credential/exchange
+          protocol|credential exchange protocol=] is initiated and the user
+          navigates from the website they're browsing to their [=holder=]
+          application. Solutions that have been used for this purpose in the
+          past include QR codes and custom URL schemes. As documented in
+          [[[presenting-credentials-on-the-web]]] and [[[custom-schemes]]],
+          those solutions have security, privacy, and accessibility concerns.
+        </p>
+        <p>
+          With adoption of digital credential technology being driven by
+          ecosystem demand and regulatory mandates, it is essential for user
+          agents to offer an alternative to these harmful technologies that is
+          easy to use for developers, is compatible with existing credential
+          exchange protocols and, most importantly, significantly increases
+          user privacy, security, and accessibility.
+        </p>
+        <p>
+          The Digital Credentials API offers the user agent the ability to
+          intermediate on behalf of the user (e.g. in the form of a
+          [=credential chooser=]) to contextualize requests and <a href=
+          "#permission-prior-to-wallet-selection">prevent immediate exposure to
+          holder applications</a>, in a way that would not be possible
+          otherwise. It also enforces certain minimum requirements on supported
```

Comment:
There is an open issue for this topic #288, but I believe currently there are no requirements for supported protocols, and no signs yet from browser implementers that they will limit support to protocols that satisfy those requirements.

Reply:
Ah yes, I was meaning to make a PR for that, so if you don't mind I'll just follow up there? But thanks for pointing it out.